[Log Point] Fix log point context menu accelerator

ravefalcon92 opened the pull request:
Fixes #7692

### Summary of Changes

* Created new keyboard shortcut(`CmdOrCtrl+Shift+Y`) for toggling conditional panel for log points
* Updated log points context menu accelerator 
* Updated ShortcutsModal with new keyboard shortcut for log points
* Changed shortcut description for `shortcuts.toggleCondPanel` from `Toggle Conditional Panel` to `Edit Conditional Breakpoint`
* Updated snapshot testing

flodolo commented on assets/panel/debugger.properties:
```diff
@@ -1023,8 +1029,12 @@ shortcuts.toggleBreakpoint=Toggle Breakpoint
-# keyboard shortcut action for toggling conditional panel keyboard
-shortcuts.toggleCondPanel=Toggle Conditional Panel
+# keyboard shortcut action for toggling conditional panel for breakpoints
+shortcuts.toggleCondPanel=Edit Conditional Breakpoint
```
Please see this document for making changes to existing strings
https://developer.mozilla.org/en-US/docs/Making_String_Changes

In short: you need a new string ID, and update all the code. I'm also not sure I understand how this fits into the PR subject, but that's more for debugger owners.

ravefalcon92 replied:
Thanks for bringing this to my attention. I'll discuss viable solutions with the debugger owners.

ravefalcon92 replied:
I appended a differentiating property to the end of `toggleCondPanel` between log points (`toggleCondPanel.logPoint`)and breakpoints (`toggleCondPanel.breakpoint`) so there is no change to existing strings. @flodolo these changes seem to be consistent with the localization best practices you linked. Let me know if this works

jasonLaster commented on src/components/Editor/index.js:
```diff
@@ -300,11 +305,16 @@ class Editor extends PureComponent<Props, State> {
+      this.toggleConditionalPanel(line, true);
+    } else {
+      this.toggleConditionalPanel(line);
+    }
```
this is a nit, but...

```js
const isLog = key === L10N.getStr("toggleLogPanel.key");
this.toggleConditionalPanel(line, isLog);
```

ravefalcon92 replied:
Fixed. Thanks for the nit

ravefalcon92 commented:
@darkwing, @jasonLaster: Wanted to get your thoughts on how we'd like to move forward with this PR.

The issue is that we currently only have one key/shortcut description for the conditional panel, but we have a conditional panel for breakpoints as well as log points and want to differentiate between the two.

My proposed solution:
~~`toggleCondPanel.key = 'CmdOrCtrl+Shift+B'`~~
`toggleCondPanel.breakpoint.key =  'CmdOrCtrl+Shift+B'`
`toggleCondPanel.logpoint.key = 'CmdOrCtrl+Shift+Y'`
~~`shorcuts.toggleCondPanel = 'Toggle Conditional Panel'`~~
`shorcuts.toggleCondPanel.breakpoint = 'Edit Conditional Breakpoint'`
`shorcuts.toggleCondPanel.logpoint = 'Edit Log Point'`

2 reasons for why I've changed the shortcut description:
1. The conditional panel is more of a term for people who are working on the debugger so it might make more sense to remove mention of conditional panel and switch to talking about breakpoints/log points.
2. Switched the shortcut description from `Toggle` to `Edit` because we are providing the users with an option to add/edit a condition or log. `Toggle` makes sense when it pertains to the conditional panel, but if we decide to remove mention of conditional panel, I think `Edit` is more clear.

darkwing commented:
Awesome, thank you!
